examples/m6-demo/README.md

```diff
@@ -0,0 +1,267 @@
+- Within the 'jane' namespace, you can see all the deployed resources (Deployment, Service, ConfigMap, Route)
+- Various DEBUG/INFO log messages are output to the console by backend and cluster-agent, corresponding to behind-the-scenes actions.
+
+Next, create a `GitOpsDeployment` resource in the jgw namespace (workspace), for our second tenant 'jgw':
+```bash
+kubectl apply -f jgw-deployment.yaml
+```
+
+Same observations as above, but notice that the deployment target is now the 'jgw' namespace. Each user has their own deployments, specified in their own namespace, deploying resources to their own namespace, backed by a shared Argo CD instance.
+
+Next, you can change fields in the `GitOpsDeployment` CR, such as the path within the target GitOps repository:
+````bash
+# Edit jgw-deployment.yaml
+vi jgw-deployment.yaml
+
+# Next, change the path field from 'dev' to 'staging':
+#
+# spec:
+#   source:
+#     path: environments/overlays/dev
+# to
+#
+# spec:
+#   source:
+#     path: environments/overlays/staging
+
+# Apply the change
+kubectl apply -f jgw-deployment.yaml
+````
+
+
+Observe that:
+- The GitOps service quickly detects the change, and updates the Argo CD Application
+- In the Argo CD UI, the corresponding Argo CD application and its resources are updated.
+- The deployed resources in 'jgw' namespace are updated.
+
+You can likewise delete the `GitOpsDeployment` CR...
+
+```
+kubectl delete -f jgw-deployment.yaml
+```
+
+... and observe that:
+- The Argo CD `Application` CR is likewise cleaned up, along with all the deployed resources.
+
+At any step in the process, you can also view the database internals:
+- Run `psql.sh` from the [managed-gitops](https://github.com/redhat-appstudio/managed-gitops) repo to automatically open *psql* utility against the PostgreSQL database.
+- Some example queries:
+    - See Argo CD Application entries: `select * from application;`
+        - _(Don't forget the semi-colon at the end of these statements!)_
+    - See completed operations: `select * from operation;` 
+    - See Argo CD instance info: `select * from gitopsengineinstance;`
+    - See deployment targets: `select * from managedenvironment;`
+    - See relationship between GitOpsDeploymentSync CRs and their corresponding Application DB entry: `select * from deploymenttoapplicationmapping;`
+    - (etc)
+- As a rule: all changes made to Argo CD are first made to the database, with the cluster-agent responsible for reconciling database state -> Argo CD state.
+
+### `GitOpsDeploymentManagedEnvironment` resource
+
+Create a `Secret` in your current namespace, containing the contents of your current kubeconfig (for simplicity sake, we use `~/.kube/config`)
+````bash
+kubectl create secret generic my-managed-environment-secret \
+       --from-file=kubeconfig=$HOME/.kube/config \
+       --type="managed-gitops.redhat.com/managed-environment" \
+       -n jane
+````
+
```

Comment:
```suggestion
Note: This command works only if your `$HOME/.kube/config` contains the configuration of your clusterbot cluster only, instead of multiple cluster contexts.
```

Reply:
@drpaneas It _should_ work as long as you specify the `apiURL` of the correct context in the `GitOpsDeploymentManagedEnvironment` CR. If you get a chance, let me know if this is the behaviour you are seeing, as otherwise it would be a bug.

I've incorporated your comment into the updated doc, but using slightly different language, describing the expected behaviour above (and then if it's buggy we can fix it to be expected).